pkg/OutputColorizer.nuspec

```diff
@@ -20,6 +20,6 @@
     </metadata>
     <files>
         <file src="net45\OutputColorizer.dll" target="lib\net45" />
-        <file src="Core\OutputColorizer.dll" target="lib\netstandard1.5" />
+        <file src="..\OutputColorizer.Core\bin\Debug\netstandard1.5\OutputColorizer.dll" target="lib\netstandard1.5" />
```

Comment:
no, it will be in `<repoRoot>\bin\OutputColorizer.Core\bin\Debug\netstandard1.5\OutputColorizer.dll`
I know it sucks that we have two nested bin folders, but there is not much we can do to customize it. All I'm specifying is that I want the output to be <repoRoot>\bin, and all the extra folders are generated by the CLI build system.